app.py

```
@@ -0,0 +1,154 @@
+        split_docs = splitter.split_documents(documents)
+
+        # projectId 검증 - 영숫자와 언더스코어만 허용
+        if not req.projectId.replace("_", "").replace("-", "").isalnum():
+            raise ValueError("잘못된 projectId 형식")
+
+        db_dir = os.path.join(BASE_DIR, f"{req.projectId}_{VECTOR_DB_NAME}")
+        if os.path.exists(db_dir):
+            shutil.rmtree(db_dir)
+
+        # 벡터 저장소에 문서 저장
+        vectorstore = Chroma.from_documents(
```

Ruff (0.13.1)

Local variable `vectorstore` is assigned to but never used

Remove assignment to unused variable `vectorstore`

(F841)